usmqe_tests/api/others/test_user.py

```diff
@@ -471,3 +471,60 @@ def test_user_add_del(valid_session_credentials, valid_normal_user_data):
     .. test_result:: 6
        :include: api/user.logout:3
     """
+
+
+@pytest.mark.xfail
+def test_user_change_username_and_email(valid_new_normal_user):
+    """@pylatest api/user.add_delete
+    API-users: add and delete
+    *************************
+
+    .. test_metadata:: author ebondare@redhat.com
+
+    Description
+    ===========
+
+    Change user's username and e-mail. New user shouldn't be created.
+    https://bugzilla.redhat.com/show_bug.cgi?id=1610660
+    """
+    auth = login(
+        "admin",
+        "adminuser")
```

Comment:
Instead of hardcoding these values and logging into WA directly, we should use `valid_session_credentials` fixture, which returns the auth object. Using `login()` directly like this is necessary only when the test covers login itself and we need to try different types of inputs for it.